plugins/commander.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
+  shortDescription: Interactively manage Kubernetes resources.
```

super nit but you don't need a trailing dot:
```suggestion
  shortDescription: Interactively manage Kubernetes resources
```